admin/class-yoast-plugin-conflict.php

```diff
@@ -182,14 +165,22 @@ protected function set_error( $plugin_section, $readable_plugin_section ) {
 		foreach ( $this->active_plugins[ $plugin_section ] as $plugin_file ) {
 
 			/* translators: %s: 'Facebook' plugin name of possibly conflicting plugin */
-			$error_message .= '<a target="_blank" class="button-primary" href="' . wp_nonce_url( 'plugins.php?action=deactivate&amp;plugin=' . $plugin_file . '&amp;plugin_status=all', 'deactivate-plugin_' . $plugin_file ) . '">' . sprintf( __( 'Deactivate %s', 'wordpress-seo' ), $this->get_plugin_name( $plugin_file ) ) . '</a> ';
+			$error_message .= '<a target="_blank" class="button-primary" href="' . wp_nonce_url( 'plugins.php?action=deactivate&amp;plugin=' . $plugin_file . '&amp;plugin_status=all', 'deactivate-plugin_' . $plugin_file ) . '">' . sprintf( __( 'Deactivate %s', 'wordpress-seo' ), WPSEO_Utils::get_plugin_name( $plugin_file ) ) . '</a> ';
 		}
 
+		// The data which will be passed in the AJAX request.
+		$conflict_data = json_encode(
+			array( 'section' => $plugin_section, 'plugins' => $this->active_plugins[ $plugin_section ] )
+		);
+		$error_message .= "<input type='hidden' name='conflict_data' value= '" . $conflict_data . "'  />";
```

Comment:
Please don't use an hidden input for this but add a `data-conflicting-plugins` attribute to the original error message.